### PR TITLE
Feature/standardized billing line item model

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** Internal Project

**This PR will result in the following new package version:** `v0.4.0`
 

This will not result in a breaking change. However, due to the feature update it would be best to highlight this as a breaking change.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

> ## Feature Updates
> - Addition of the `stripe__line_item_enhanced` model. This model constructs a comprehensive, denormalized analytical table that enables reporting on key revenue, subscription, customer, and product metrics from your billing platform. It’s designed to align with the schema of the `*__line_item_enhanced` model found in Recurly, Recharge, Stripe, and Zuora, offering standardized reporting across various billing platforms. To see the kinds of insights this model can generate, explore example visualizations in the [Fivetran Billing Model Streamlit App](https://fivetran-billing-model.streamlit.app/). Visit the app for more details.
> 
> ## Under the Hood:
> - Included auto-releaser GitHub Actions workflow to automate future releases.
> - Incorporated the new `fivetran_utils.drop_schemas_automation` macro into the end of each Buildkite integration test job.
> - Updated the pull request [templates](/.github).

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt run –full-refresh && dbt test
- [X] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked, tagged, and properly assigned
- [X] All necessary documentation and version upgrades have been applied
    
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [X] BuildKite integration tests are passing
- [X] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Streamlit app creation highlights the validation efforts. Additionally, consistency test was created and is expected to fail since the new model does not exist in prod yet. This will be leveraged for future updates.

### If you had to summarize this PR in an emoji, which would it be?
 
💬

### CODE REVIEW

**Review by fivetran-reneeli — APPROVED** [review]
Looks good!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+- Addition of the `recurly__line_item_enhanced` model. This model constructs a comprehensive, denormalized analytical table that enables reporting on key revenue, subscription, customer, and product metrics from your billing platform. It’s designed to align with the schema of the `*__line_item_enhanced` model found in Recurly, Recharge, Stripe, Shopify, and Zuora, offering standardized reporting across various billing platforms. To see the kinds of insights this model can generate, explore example visualizations in the [Fivetran Billing Model Streamlit App](https://fivetran-billing-model.streamlit.app/). Visit the app for more details.
+  -  This model is currently disabled by default. You may enable it by setting the `recurly__standardized_billing_model_enabled` as `true` in your `dbt_project.yml`.
+
+## Under the Hood:
+- Added consistency test within integration_tests for the `recurly__line_item_enhanced` model.
+- Included auto-releaser GitHub Actions workflow to automate future releases.
 - Incorporated the new `fivetran_utils.drop_schemas_automation` macro into the end of each Buildkite integration test job.
 - Updated the pull request [templates](/.github).
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:13]: I would just remove the relative link since it will be broken in the release body.

**Review by fivetran-catfritz — COMMENTED** [review]
Tiny changelog suggestion. Also looking at the commit history I'm not seeing we regenned the docs after disabling the model by default, so maybe we need to regen docs again.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md:13]: oh good catch to remove the relative paths!

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Pull request merged**